projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader-config.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Inject, Injectable, Optional } from '@angular/core';
+import {
+  DEFAULT_NGX_SKELETON_LOADER_CONFIG,
+  NgxSkeletonLoaderConfig,
+  NGX_SKELETON_LOADER_CONFIG,
+} from './ngx-skeleton-loader-config.types';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class NgxSkeletonLoaderConfigService {
```

Comment:
Forgive me for intrude your review into this PR, but unless I'm missing something I think this file isn't even necessary as you already have the `InjectionToken` `NGX_SKELETON_LOADER_CONFIG`.

So in `NgxSkeletonLoaderComponent` you can have this:

```ts
constructor(@Inject(NGX_SKELETON_LOADER_CONFIG) @Optional() private readonly config?: NgxSkeletonLoaderConfig) {
  // ... 
}
```

---

and we can use it like this in `@NgModule` level:

```
@NgModule({
  imports: [NgxSkeletonLoaderModule.forRoot(config)],
  // ... or:
  // imports: [NgxSkeletonLoaderModule],
  // providers: [{ provide: NGX_SKELETON_LOADER_CONFIG, useValue: config }],
})
```

or at `@Component` level:

```
@Component({
  providers: [{ provide: NGX_SKELETON_LOADER_CONFIG, useValue: config }],
})
```

Reply:
Hey there! Thank you for the review on your side! I also thought the same about the current implementation behind redundant with this service. I will take a look at moving everything to the one you mentioned hereabove.